com.fightcade.Fightcade.desktop

```diff
@@ -0,0 +1,12 @@
+[Desktop Entry]
```

Comment:
The reason this isn't upstream is because the application isn't _installed_ exactly, they provide a desktop file via a shell script that echos it into `~/.local/share/applications`. Same for the other desktop file.

I am just providing desktop files for my wrappers.